.eslintrc.json

```diff
@@ -16,6 +19,14 @@
     "setInterval": true,
     "clearInterval": true
   },
+  "settings": {
+    "jsdoc": {
```

Comment:
Should we have updates to README or CONTRIBUTING regarding the typescript?  Is the build/inclusion process any different?

Reply:
Good catch. There should be some updates to the README. Maybe a "migrating from 3.x" section. I will add it.

Reply:
@swiftone Please take a look at changes added to README